### DISCUSSION TITLE
Getting task attached to a different loop error while starting zeebe client

### DISCUSSION THREAD

**Discussion by aliakbarnath** [camunda-community-hub/pyzeebe#379] [Q&A]
Getting this error while starting zeebe client

RuntimeError: Task <Task pending name='Task-1' coro=<main() running at /home/k10057/Downloads/camunda-zeebe/zeebe_client.py:16> cb=[_run_until_complete_cb() at /usr/local/lib/python3.9/asyncio/base_events.py:184]> got Future <Task pending name='Task-2' coro=<UnaryUnaryCall._invoke() running at /Downloads/camunda-zeebe/venv/py/lib/python3.9/site-packages/grpc/aio/_call.py:568>> attached to a different loop

**Comment by dimastbk** [comment]
See warning https://camunda-community-hub.github.io/pyzeebe/worker_quickstart.html#create-and-start-a-worker
